src/main/java/com/jaspersoft/jasperserver/jaxrs/client/dto/settings/DateTimeSettings.java

```diff
@@ -0,0 +1,67 @@
+package com.jaspersoft.jasperserver.jaxrs.client.dto.settings;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Created by tetiana.iefimenko on 7/22/2015.
+ */
+public class DateTimeSettings {
+
+    private  Datepicker datepicker;
+    private Timepicker timepicker;
+
+    public DateTimeSettings() {
+    }
+
+    public DateTimeSettings(DateTimeSettings other) {
+        this.datepicker = other.datepicker;
```

Comment:
Original datepicker and timepicker instance shouldn't be assigned to a clone. Please use their cloning constructor to avoid change by reference